devserver/devserver/BUILD.bazel

```diff
@@ -3,6 +3,9 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 go_library(
     name = "go_default_library",
     srcs = ["devserver.go"],
+    deps = [
+        "@io_bazel_rules_go//go/tools/bazel:go_default_library",
```

Comment:
@devversion is this the change which required the gazelle directive above?

Reply:
Yeah that's one affected case. There is also one in `devserver/BUILD.bazel`.